content/blog/2016/2016-01-27-jenkins-world-call-for-papers.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Jenkins World 2016: Call For Papers Is Open!"
+tags:
+- jenkins world
+- event
+- jenkins
+author: alyssat
+---
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016,180,float="right",link="https://jenkins-cfp.herokuapp.com/events/jenkins-world-2016"]
+
+Planning is underway for the annual Jenkins Users Conference, to be held from

```

Comment:
"*the*" annual conference?  I'm sure there will be other Jenkins conferences this year, even if CloudBees did decide to use the specific "Jenkins User Conference" name in the past (without trademark request) ;)